proxy/src/main/java/net/md_5/bungee/util/ChatComponentTransformer.java

```diff
@@ -51,7 +51,7 @@ public static ChatComponentTransformer getInstance()
      */
     public BaseComponent[] transform(ProxiedPlayer player, BaseComponent... component)
     {
-        if ( component == null || component.length < 1 )
+        if ( component == null || component.length < 1 || component[0] == null  )
```

Comment:
I think better code would be:

`component == null ||  component.length < 1 || ( component.length == 1 && component[0] == null )`
Doing this validation in this method isn't ideal, but at least this way we don't allow things like:
`transform(player, null, <component>` to accidentally become valid

Reply:
I think you mean:
transform(player, <component>, null

Yeah, i agree.